overlay.d/05core/usr/lib/dracut/modules.d/40ignition-ostree/ignition-ostree-transposefs.sh

```diff
@@ -139,13 +139,20 @@ should_autosave_rootfs() {
         return
     fi
     local agcount
+    # We want to run xfs_info on the unmounted filesystem, because actually just
+    # even mounting an XFS filesystem that has grown an excessive number of allocation groups
+    # can be very slow.
```

Comment:
This comment confuses me. You didn't change the following line of code so you were trying to add context? Is the filesystem unmounted or are you saying we want to operate on an unmounted filesystem in the future?

Maybe this would help:

```suggestion
    # We run xfs_info on the unmounted filesystem, because actually just
    # even mounting an XFS filesystem that has grown an excessive number of allocation groups
    # can be very slow.
```